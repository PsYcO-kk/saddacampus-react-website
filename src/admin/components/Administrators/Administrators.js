import React from 'react';
import { Link } from 'react-router-dom';
import { Button, Table } from 'reactstrap';

const Administrators = () => (
	<div>
		<Link to="/admin/administrators/create" className="nav-link">
			<Button color="primary" size="lg" block>Create New Administrator</Button>
		</Link>
		<br />
		<Table striped>
			<thead>
				<tr>
					<th>#</th>
					<th>Admin ID</th>
					<th>Username</th>
					<th>Password</th>
					<th>Email</th>
					<th>Role</th>
				</tr>
			</thead>
			<tbody>
				<tr>
					<th scope="row">1</th>
					<td>sadd4kfue954ji4nk014</td>
					<td>saddacampussuper</td>
					<td>Campusjoy69</td>
					<td>saddacampus@gmail.com</td>
					<td>Super</td>
				</tr>
			</tbody>
		</Table>
	</div>
);

export default Administrators;
