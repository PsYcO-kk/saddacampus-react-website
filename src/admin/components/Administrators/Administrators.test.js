import React from 'react';
import { shallow } from 'enzyme';
import Administrators from './Administrators';

test('should render Administrators component without crashing', () => {
	const wrapper = shallow(<Administrators />);
	expect(wrapper).toMatchSnapshot();
})
