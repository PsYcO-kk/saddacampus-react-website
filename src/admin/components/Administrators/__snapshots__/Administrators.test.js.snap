// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Administrators component correctly 1`] = `
<div>
  <Link
    className="nav-link"
    replace={false}
    to="/administrators/create"
  >
    <Button
      block={true}
      color="primary"
      size="lg"
      tag="button"
    >
      Create New Administrator
    </Button>
  </Link>
  <br />
  <Table
    responsiveTag="div"
    striped={true}
    tag="table"
  >
    <thead>
      <tr>
        <th>
          #
        </th>
        <th>
          Admin ID
        </th>
        <th>
          Username
        </th>
        <th>
          Password
        </th>
        <th>
          Email
        </th>
        <th>
          Role
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <th
          scope="row"
        >
          1
        </th>
        <td>
          sadd4kfue954ji4nk014
        </td>
        <td>
          saddacampussuper
        </td>
        <td>
          Campusjoy69
        </td>
        <td>
          saddacampus@gmail.com
        </td>
        <td>
          Super
        </td>
      </tr>
    </tbody>
  </Table>
</div>
`;
